actors/power/src/state.rs

```diff
@@ -68,6 +68,9 @@ pub struct State {
     /// Number of miners having proven the minimum consensus power.
     pub miner_above_min_power_count: i64,
 
+    pub ramp_start_epoch: i64,
+    pub ramp_duration_epochs: u64,
```

Comment:
Please document these two fields, referencing FIP-0081 and describing their specific intent.

Reply:
Done. I wish the other fields had documentation. :)